extensions/stackdriver/stackdriver.cc

```diff
@@ -223,10 +224,7 @@ bool StackdriverRootContext::configure(size_t configuration_size) {
   }
   google::protobuf::Struct node;
   getLocalNodeMetadata(&node);
-  if (!::Wasm::Common::extractLocalNodeFlatBuffer(&local_node_info_, node)) {
-    logWarn("cannot extract local node metadata");
-    return false;
-  }
+  local_node_info_ = ::Wasm::Common::extractNodeFlatBufferFromStruct(node);
```

Comment:
Shall we put this conversion also into `getLocalNodeMetadata`? So that it could just a detachedbuffer. This conversion confuses a bit, until I fully read the function.

Reply:
Done.